assets/prompts/assistant_system_prompt.hbs

```diff
@@ -27,13 +27,14 @@ If appropriate, use tool calls to explore the current project, which contains th
 - `{{root_name}}`
 {{/each}}
 
+- Bias towards not asking the user for help if you can find the answer yourself.
 - When providing paths to tools, the path should always begin with a path that starts with a project root directory listed above.
+- Before you read or edit a file, you must first find the full path. DO NOT ever guess a file path!
+{{# if (hasTool 'grep') }}
```

Comment:
Can we make the function name snake_case?

```suggestion
{{# if (has_tool 'grep') }}
```

Reply:
Sure, done!